delphi/app/routes.py

```diff
@@ -9,9 +9,64 @@ def updateSensorData():
         sensorId = int(data.get('id'))
         count = int(data.get('count'))
 
-        # todo: store sensor data
+        # TODO: store sensor data
         return '', 204
     except:
         pass
 
     return 'Invalid Request Syntax', 400
+
+@app.route('/enable/<room_id>', methods=['PUT'])
```

Comment:
It looks like room id is expected to be an integer. To force it, you can use: /enable/<int:room_id>, so you don't need to cast to int later on and I believe flask won't allow that route to go through if it isn't an int.